src/ert/scheduler/scheduler.py

```diff
@@ -92,6 +85,9 @@ def __init__(
         ee_token: Optional[str] = None,
```

Comment:
These `ee_*` class variables can be removed I suppose.

Reply:
yes, but `_update_jobs_json` needs to be moved to (based_run_model), which I thought could be a different PR?